src/traces/contour/make_color_map.js

```diff
@@ -21,6 +21,11 @@ module.exports = function makeColorMap(trace) {
         nc = Math.floor((end - start) / cs) + 1,
         extra = contours.coloring === 'lines' ? 0 : 1;
 
+    if(!isFinite(cs)) {
```

Comment:
why? 

P.S. I apologize for being lazy and not reviewing this PR commit by commit.